DigitalLearningSolutions.Web/Controllers/SignpostingController.cs

```diff
@@ -0,0 +1,77 @@
+﻿namespace DigitalLearningSolutions.Web.Controllers
+{
+    using System.Threading.Tasks;
+    using DigitalLearningSolutions.Data.ApiClients;
+    using DigitalLearningSolutions.Data.DataServices.UserDataService;
+    using DigitalLearningSolutions.Data.Services;
+    using DigitalLearningSolutions.Web.Helpers;
+    using DigitalLearningSolutions.Web.ViewModels.Signposting;
+    using Microsoft.AspNetCore.Authorization;
+    using Microsoft.AspNetCore.Mvc;
+
+    [Authorize(Policy = CustomPolicies.UserOnly)]
+    [Route("Signposting")]
+    public class SignpostingController : Controller
+    {
+        private readonly IActionPlanService actionPlanService;
+        private readonly ILearningHubApiClient learningHubApiClient;
+        private readonly IUserDataService userDataService;
+        private readonly IUserService userService;

```

Comment:
I think we should final get around to making a pass through method in UserService for get delegate by ID, means we can get rid of the data service dependency from controllers

Reply:
Added the mentioned method and a method for the new updating of the dismissal status. Haven't applied them anywhere other than the new controller though.